src/codeu/chat/client/core/Controller.java

```diff
@@ -112,4 +108,98 @@ public ConversationHeader newConversation(String title, Uuid owner)  {
 
     return response;
   }
+
+  @Override
+  public void newUserInterest(String name, Uuid owner) {
```

Comment:
these functions shouldn't return void. See the other functions, the controller should return the delta that it caused to the backend, e.g. newUser returns the User object that it added, newMessage returns a Message object and so on